.travis.yml

```diff
@@ -2,13 +2,23 @@ language: bash
 services: docker
 
 env:
-  image=eclecticiq/package
+  global:
+    image=eclecticiq/package
+  matrix:
+    - version=1.11.0 latest=true
+    - version=1.10.2
+    - version=1.10.1
+    - version=1.10.0
 
-script:
-  - travis_retry docker build --tag "$image:latest" .
-
-after_success:
-  - if [ "$TRAVIS_BRANCH" == "master" ]; then
-    docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
-    docker push $image;
-    fi
+jobs:
+  include:
+    - stage: build
+      script: travis_retry docker build --build-arg FPM_VERSION=$version --tag "$image:$version" .
+    - stage: tag latest
+      script: docker tag "$image:$version" "$image:latest"
+      env: latest=true
```

Comment:
travis-ci's docs makes it unclear, but I think this sets the environment variable rather than conditionally runs the stage...

Reply:
Yeah, but it didn't work either as stages are run in different VMs.